CONTRIBUTING.md

```diff
@@ -125,6 +125,130 @@ When you introduce changes that bump suppported Telegram Bot API version (e.g. 6
 - Change TBA version in `…https://img.shields.io/badge/API%20coverage…` line in [crates/teloxide-core/README.md](crates/teloxide-core/README.md) file
 - Change TBA version in `…https://img.shields.io/badge/API%20coverage…` line in [README.md](README.md) file
 
+### Adding a new method
+
+It should be noted first that TBA updates that contain new methods are usually divided between developers in the [teloxide_dev](https://t.me/teloxide_dev) chat, or in the github issues for new TBA versions. If you want to contribute a TBA update, contact other developers first, the work that you want to do may already be done. 
+
+#### Step 1:
+
+Add the method and its info to `crates/teloxide-core/src/schema.ron` file.
+
+For example, lets add a `createChatInviteLink`. Look at the [TBA documentation](https://core.telegram.org/bots/api#createchatinvitelink).
```

Comment:
I think it's better to specify the TBA version here (…lets add a `createChatInviteLink` from TBA 7.9…) and use [Web Archive](https://web.archive.org/), because we often do TBA updates with a delay of a few versions, and the added methods may already be updated in TBA. So Web Archive here helps a lot with snapshots of documentation over time like this: https://web.archive.org/web/20240816155327/https://core.telegram.org/bots/api#createchatinvitelink